polish the release notes (#352)
nim-lang/website: jekyll/_posts/2022-12-21-version-20-rc.md

- `shallowCopy` and `shallow` are removed for ARC/ORC. Use `move` when possible or combine assignment and
`sink` for optimization purposes.

> - The experimental `nimPreviewDotLikeOps` switch is going to be removed or deprecated because it was bad designed.

Reviewer: ```suggestion
- The experimental `nimPreviewDotLikeOps` switch is going to be removed or deprecated because it didn't fullfill its promises.
```

```

The types `E1` and `E2` share the names `value1` and `value2`. These are overloaded and the usual overload disambiguation

> is used so that the `E1` or `E2` prefixes can be left out in many cases. Those features are most beneficial for independently developed libraries.

Reviewer: ```suggestion
is used so that the `E1` or `E2` prefixes can be left out in many cases. These features are most beneficial for independently developed libraries.
```
